src/sentry/api/serializers/models/group.py

```diff
@@ -187,18 +187,47 @@ def get_attrs(self, item_list, user):
             group__in=item_list,
         )}
 
-        resolutions = {
-            i[0]: i[1:]
-            for i in GroupResolution.objects.filter(
-                group__in=item_list,
-            ).values_list(
-                'group',
-                'type',
-                'release__version',
-                'actor_id',
-            )
-        }
-        actor_ids = set(r[-1] for r in six.itervalues(resolutions))
+        resolved_item_list = [i for i in item_list if i.status == GroupStatus.RESOLVED]
+        if resolved_item_list:
+            release_resolutions = {
+                i[0]: i[1:]
+                for i in GroupResolution.objects.filter(
+                    group__in=resolved_item_list,
+                ).values_list(
+                    'group',
+                    'type',
+                    'release__version',
+                    'actor_id',
+                )
+            }
+
+            # due to our laziness, and django's inability to do a reasonable join here
+            # we end up with two queries
+            commit_results = list(Commit.objects.extra(
+                select={
+                    'group_id': 'sentry_grouplink.group_id',
+                },
+                tables=['sentry_grouplink'],
+                where=[
+                    'sentry_grouplink.linked_id = sentry_commit.id',
+                    'sentry_grouplink.group_id IN ({})'.format(
+                        ', '.join(six.text_type(i.id) for i in resolved_item_list)),
```

Comment:
Are we able to safely assume that `item_list` does not contain arbitrary user data?

Reply:
Yeah - you mean in that item.id is safe?

Reply:
yes. I asked because its being smushed into a SQL snippet.